osvr/RenderKit/osvr_display_configuration.h

```diff
@@ -132,6 +132,9 @@ class OSVRDisplayConfiguration {
     std::vector<float> const& getDistortionPolynomalBlue() const;
     ///@}
 
+    /// Returns the desired number of triangles in the constructed distortion mesh.
+    int OSVR_RENDERMANAGER_EXPORT getDesiredDistortionTriangleCount() const;
```

Comment:
I was following the rest of the declarations in this file, all of which have it after the return type.  No compilers complain about this placement.  If you want all entries in this file changed to another ordering, that can be done.